packages/form-js-viewer/src/util/index.js

```diff
@@ -133,10 +132,3 @@ export function getSchemaVariables(schema) {
   // remove duplicates
   return Array.from(new Set(variables));
 }
-
-
```

Comment:
Moved this out and referred to the one in feel, that way we only have to change this in one place and won't get weird bugs.